src/app/layout/navbar/language-button/language-button.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+
+import { TranslateService } from '@ngx-translate/core';
+import { Observable, Subject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
```

Comment:
Do not provide services in the root. Add an array of providers in the layout module, and put this there.